boards/feather-nrf52840/Makefile.dep

```diff
@@ -1,5 +1,12 @@
-# Use Segger's RTT by default for stdio on this board
-DEFAULT_MODULE += stdio_rtt
+# Provide stdio over USB by default
+# This is a temporary solution until management of stdio is correctly
+# handled by the build system
+DEFAULT_MODULE += stdio_cdc_acm
+
+ifeq (,$(filter stdio_cdc_acm,$(DISABLE_MODULE)))
+  # Default auto-initialization of usbus for stdio other USB
+  DEFAULT_MODULE += auto_init_usbus
+endif
```

Comment:
Do all `stdio_cdc_acm` need `auto_init_usbus`? If so it should go into `stdio.inc.mk`, and just:

```
ifneq (,$(filter stdio_cdc_acm,$(USEMODULE)))
 USEMODULE += auto_init_usbus
endif
```

Reply:
IMHO, this is another PR as other boards would need to be addressed. But I agree on the idea of moving this in stdio.inc.mk.

Reply:
No board declares `DEFAULT_MODULE += auto_init_usbus`

Reply:
This is just semi-related (but related to the build failure of `tests/minimal`), afaiu `stdio.inc.mk` should also have 

```make
ifneq (,$(filter stdio_null,$(USEMODULE)))
  DISABLE_MODULE += $(filter-out stdio_null,$(STDIO_MODULES))
endif
```

Reply:
No but some use `USEMODULE += auto_init_usbus`

Reply:
But `auto_init_usbus` is only needed if `stdio_cdc_acm` is used. Otherwise, a dependency is pulled in that is not needed.

Reply:
> IMHO, this is another PR as other boards would need to be addressed. But I agree on the idea of moving this in stdio.inc.mk.

Can somebody please do that? I don't have the time atm (and not enough experience with boards) at hand to shepherd such an effort.

Reply:
I would base this PR then on that one.

Reply:
#13703 would handle it.

Reply:
> I would base this PR then on that one.

Done.